Utils/helper_functions.py

```diff
@@ -0,0 +1,60 @@
+# functions to find files, get median,
+
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+
+# function to open a file and return the data in a pandas dataframe
+def open_file(file_path):
```

Comment:
Consider providing a more opinionated name for this function to help differentiate from the more generalized `open` functionality in Python. For example: `read_csv_as_df`.